docs/docs/how-to-deploy.mdx

```diff
@@ -136,6 +138,87 @@ The `<custom_image_tag>` should reference how this image will be different from
 example, you could version or date your tags, as well as create different tags that have different code for production
 and development servers. You should create a new tag any time you update your code and want to re-deploy it.
 
+#### Automating your Action Server Image Builds
+
+In addition to a manual step to create a new Action Server Image. You can use [Rasa Action Server GitHub Action](https://github.com/RasaHQ/action-server-gha) to automate image builds.
+If GitHub Actions are new to you it might be helpful to get familiar with [GitHub Actions Documentation](https://docs.github.com/en/actions).
```

Comment:
```suggestion
If GitHub Actions are new to you, it might be helpful to get familiar with [GitHub Actions Documentation](https://docs.github.com/en/actions).
```